test/functional/setup.js

```diff
@@ -98,8 +118,15 @@ before(function () {
                 var fixturePath    = path.join(path.dirname(caller()), fixture);
                 var skipJsErrors   = opts && opts.skipJsErrors;
                 var quarantineMode = opts && opts.quarantineMode;
+                var skipOption     = opts && opts.skip;
+                var onlyOption     = opts && opts.only;
+
+                var connections = browsersInfo.filter(function (browserInfo) {
+                    var only = onlyOption ? onlyOption.indexOf(browserInfo.settings.alias) > -1 : true;
+                    var skip = skipOption ? skipOption.indexOf(browserInfo.settings.alias) < 0 : true;
 
-                var connections = browsersInfo.map(function (browserInfo) {
+                    return only && skip;

```

Comment:
it's a strange combination of `only and skip`. Why we return `true` is `skip` is `true`? We should return `only && !skip` here as it was before